<ui version="4.0" >
 <class>NewImageSelectPage</class>
 <widget class="QWidget" name="NewImageSelectPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>432</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Select images you'd like to be in the galleries.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="deselectAllButton" >
     <property name="text" >
      <string>&amp;Deselect all</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="selectAllButton" >
     <property name="text" >
      <string>&amp;Select all</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>181</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QFrame" name="progressFrame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QLabel" name="outputLabel" >
        <property name="text" >
         <string>Searching for images... Current image %1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="3" row="1" column="0" >
    <widget class="QListWidget" name="imageList" >
     <property name="dragDropMode" >
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="iconSize" >
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="textElideMode" >
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="movement" >
      <enum>QListView::Static</enum>
     </property>
     <property name="flow" >
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0" >
      <bool>true</bool>
     </property>
     <property name="resizeMode" >
      <enum>QListView::Adjust</enum>
     </property>
     <property name="gridSize" >
      <size>
       <width>140</width>
       <height>200</height>
      </size>
     </property>
     <property name="viewMode" >
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes" >
      <bool>false</bool>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>selectAllButton</sender>
   <signal>clicked()</signal>
   <receiver>imageList</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>522</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>236</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deselectAllButton</sender>
   <signal>clicked()</signal>
   <receiver>imageList</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>559</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>367</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
